lib/index.js

```diff
@@ -33,10 +34,15 @@ co(function * () {
 		fly.emit('fly_run', {path: fly.file}).start(tasks)
 	}
 }).catch(function (e) {
-	if (e.code === 'ENOENT') {
-		utils.error('No Flyfile? See the Quickstart guide → git.io/fly-quick')
```

Comment:
In which case this error could be fired, because we added `flyfile` not found handling here https://github.com/flyjs/fly/commit/8e249a07dd0f021f345aa0b1f6eebd358ecf57fa ?